app/presenters/document/field_group_presenter.rb

```diff
@@ -106,23 +107,16 @@ def section_field_subsection(section)
           if section[:url]
             item[:url] = field_value(section[:url])
           elsif section[:search_field]
-            item[:url] = section_field_search_path(val, section[:search_field], section[:quoted])
+            item[:url] = if entity_section?(section) && Rails.application.config.x.enable.enable_entity_page
```

Comment:
This breaks the linked (to search) display of dcCreator links where values do not come from `agents`.

Compare for example the output of http://www.europeana.eu/portal/en/record/9200507/Reading_Europe_303470.html?debug=json with http://localhost:3000/portal/en/record/9200507/Reading_Europe_303470.html?debug=json when this branch is run locally. All creators (Marinos Ioannides, Cyprus University of Technology) on that record need to continue to be links to searches.